README.md

```diff
@@ -0,0 +1,72 @@
+# ATD Bond Reporting
+
+ETL that is the backend of the ATD/PWD Bond Spending Dashboard in Power BI. 
+
+***
+
+## microstrategy_to_s3.py
+
+This script runs and then downloads a report from Microstrategy and then uploads it as a csv file in a S3 bucket. 
+
+### Config
+
+The `REPORTS` dict (`report_name`:`report_id`) at the top of this file stores what reports can be extracted. 
+
+### Power BI
+
+The purpose of this script is to make it easier to access Microstrategy reports in external applications like Power BI. The URL for all objects in the S3 bucket is default to publicly accessible. bond_data.py and bond_calculations.py build off of this. 
```

Comment:
This is a cool approach! Would you be up for making a lil [excalidraw](https://excalidraw.com/) that shows the basic data flow? You can add the diagram to the readme and commit the excalidraw JSON as well.

Reply:
> S3 bucket is default to publicly accessible

This is not needed anymore since Power BI is not accessing the files directly from S3 anymore. I think this is a cost savings for us if we do this?

Reply:
i believe we play for s3 data egress — so keeping the buckets private is the way to go!

Reply:
@johnclary I've set the two buckets this script was touching to private and edited this script to use boto3 to get the data from S3 instead 👍

Reply:
nice—looks great!